scripts/pipeline/orthology_benchmark.py

```diff
@@ -175,6 +177,64 @@ def get_core_names(species_names: List[str], host: str, port: int, user: str) ->
     return core_names
 
 
+def get_gtf_file(core_name: str, source_dir: str, target_dir: str) -> None:
+    """Finds and copies a GTF file for a specified core db.
+
+    Note:
+        The code utilises the structure of `production/ensemblftp` or MC's personal dir
+        to speed up file search.
+
+    Args:
+        core_name: Core db name.
+        source_dir: Path to the directory containing subdirectories with GTF files.
+            `production/ensemblftp` or MC's `nobackup/release_dumps`.
+        target_dir: Path to the directory where the GTF file will be copied.
+
+    Raises:
+        RuntimeError: If command to copy a file fails for any reason.
+
+    Warns:
+        UserWarning: If the GTF file was not found.
```

Comment:
Good point. I made local changes so in case the function looks for the GTF file of e.g. a `release-53` core, and `release-53/` doesn't exist, it raises a warning that the GTF file was not found. Will push later.